index.html

```diff
@@ -2,46 +2,47 @@
 <html>
   <head>
     <meta charset="utf-8"/>
-    <meta name="description" content="This is a boilerplate for building websites."/>
-    <title>Boilerplate</title>
-    <link rel="stylesheet" href="css/style.css" type="text/css"/>
+    <meta name="description" content="Change this!"/>
+    <title></title>
     <link rel="stylesheet" href="css/reset.css" type="text/css"/>
+    <link href="http://fonts.googleapis.com/css?family=Roboto:400,300,500,700" rel="stylesheet" type="text/css">
+    <link rel="stylesheet" href="css/style.css" type="text/css"/>
 
   <body>
-    <div container="wrapper">
+    <div class="wrapper">
 
-      <div container="header">
-        <img src="#" href="#" alt="logo">
-        <div container="nav">
+      <div class="header">
+        <img src="http://placehold.it/300x300/cdcdcd" href="#" alt="logo">
+        <div class="nav">
           <ul>
-            <li>About</li>
-            <li>Menu</li>
-            <li>Blog</li>
-            <li>Contact</li>
+            <li><a class="about" href="#">About</a></li>
+            <li><a class="blog" href="#">Blog</a></li>
+            <li><a class="contact" href="#">Contact</a></li>
+            <li><a class="menu" href="#">Menu</a></li>
           </ul>
         </div>
      </div>
 
-     <div container="main">
-       <div container="aside">
-         <img src="#" href="#" alt="picture">
+     <div class="main clearfix">
+       <div class="aside">
+         <img src="http://placehold.it/250x250/cdcdcd" href="#" alt="picture">
          <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit.</p>
        </div>
-       <div container="section">
+       <div class="section">
          <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
       </div>
     </div>
 
-    <div container="footer">
-      <div container="social">
+    <div class="footer">
+      <div class="social">
         <img src="#" href="#"> <img src="#" href="#"> <img src="#" href="#">
       </div>
     </div>
 
   </div>
 
-    <script src="js/script.js"></script>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
+    <script src="js/script.js"></script>
 
 
   </body>
```

Comment:
Keep up the good work! Start ironing out some of the details and filling it in with real content. Whenever you're ready, hit the big green "Merge" button at the bottom.